maths/decimal_convert.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @function decimalConvert
+ * @description Convert the binary to decimal.
+ * @param {string} binary - The input binary
+ * @return {number} - Decimal of binary.
+ * @see [DecimalConvert](https://www.programiz.com/javascript/examples/binary-decimal)
+ * @example decimalConvert(1100) = 12
```

Comment:
```suggestion
 * @example decimalConvert("1100") = 12
```

I think one example is enough.